queue/queue.go

```diff
@@ -0,0 +1,34 @@
+package queue
+
+import (
+	"runtime"
+
+	"github.com/Ptt-official-app/go-openbbsmiddleware/configutil"
+
+	"github.com/golang-queue/queue"
+)
+
+const configPrefix = "go-openbbsmiddleware:queue"
+
+var client *queue.Queue
+
+func Start() error {
+	queueSize := configutil.SetIntConfig(configPrefix, "QUEUE_SIZE", 4096)
```

Comment:
I prefer the original queue/config.go queue/config_util.go setup
In that I can easily know what are the configurable global variables.

I found that queue.InitConfig() actually missed in main_init.go
we should add queue.InitConfig in main_init.go

Reply:
(We can refer to types/00-config.go, types/config_util.go, types/config.go and see how the config-variables are setup in this project～)

Reply:
@chhsiao1981 revert the queue/config.go and queue/config_util.go file

Reply:
hi @appleboy 

Thanks for your reply～
I saw your comments～
but I didn't see the changes yet～

Reply:
@chhsiao1981 I forgot push changes. XD

Reply:
@chhsiao1981 All done.

Reply:
/lgtm